guide/how-it-works/sign-in-with-bitcoin.md

```diff
@@ -0,0 +1,161 @@
+---
+layout: guide
+title: Sign in with bitcoin
+description: An introduction to how lightning wallets can support authentication and payments with external applications and web services.
+nav_order: 9
+parent: How it works
+permalink: /guide/how-it-works/sign-in-with-bitcoin/
+main_classes: -no-top-padding
+image: https://bitcoin.design/assets/images/guide/how-it-works/sign-in-with-bitcoin/sign-in-with-bitcoin-preview.jpg
+---
+
+<!--
+
+Editor's notes
+
+This page provides a top-level overview of the interactions around signing in
+and making payments on a web application via a lightning wallet.
+
+-->
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/sign-in-with-bitcoin/sign-in-with-bitcoin.jpg"
+   retina = "/assets/images/guide/how-it-works/sign-in-with-bitcoin/sign-in-with-bitcoin@2x.jpg"
+   mobile = "/assets/images/guide/how-it-works/sign-in-with-bitcoin/sign-in-with-bitcoin-mobile.jpg"
+   mobileRetina = "/assets/images/guide/how-it-works/sign-in-with-bitcoin/sign-in-with-bitcoin-mobile@2x.jpg"
+   alt-text = "Hand cursor hovering over a big 'Sign in' button"
+   caption = ""
+   width = 1600
+   height = 600
+   layout = "full-width"
+%}
+
+# Sign in with bitcoin
+{:.no_toc}
+
+---
+
+<div class="glossary-toc" markdown="1">
+* Table of contents
+{:toc}
+</div>
+
+---
+
+Identity is at the core of bitcoin, as it is a requirement to create what we refer to as ownership. Bitcoin is assigned to addresses. Addresses are generated from public-private key pairs. Owning bitcoin means being solely in control of these private keys. See [public-key cryptography](https://en.wikipedia.org/wiki/Public-key_cryptography)) for more information on this topic.
+
+The cryptography involved in this can also be used to generate public keys for other purposes, like a unique identifier that websites and applications can use for their account systems. This is similar to approaches like [Passkeys](https://developer.apple.com/passkeys/), which also look at replacing email & password combos with unique identifiers. [Decentralized identities](https://en.wikipedia.org/wiki/Decentralized_identifier) (DID) are another emerging approach.
+
+In the context of bitcoin, this allows users to sign in to services with their wallets and also use this connection to make low-friction payments.
+
+{% include tip/open.html color="blue" icon="info" label="All eyes on lightning" %}
+
+On this page, we only focus on lightning wallets, as they lend themselves much better to this type of user experience.
```

Comment:
Curios, why is this so? Both on-chain / lightning use public-key cryptography.

Reply:
From a practical perspective, I can use webLN and LNURL-Auth today and I don't think any on-chain wallets do this (AFAIK). 

The other point is that lightning wallets are better for use cases around micropayments and allowances. Having to wait 10 minutes for a of payment confirmation is bit of a deal breaker. Bit of the same logic we have around the daily spending wallet.

Overall I am assuming that we will have to keep updating this page as things evolve. I heard that more webLN capabilities are bring discussed, as well as a [webBTC spec](https://balls.dev). I believe @johnsBeharry is involved in this.